types/reach__router/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "use-default-type-parameter": false
```

Comment:
Please use a comment flag if you truly need to disable a rule. It helps track where the rule is abused:
https://github.com/DefinitelyTyped/DefinitelyTyped#linter